### DISCUSSION TITLE
Is there any way to run the overlay above a fullscreened game?

### DISCUSSION THREAD

**Discussion by Noobiangr** [baffler/Transparent-Twitch-Chat-Overlay#298] [Q&A]
I tried to stream Hunt Showdown but because of the game engine,the game performs very bad when in borderless-windowed mode,so i have to run it in fullscreen without being able to use the overlay..
Any ideas? Can the app run above the game?

**Comment by IAmAProfessionalGamer** [comment]
You can try Borderless Window mode and see if that helps, then it acts like a window.

**Comment by Noobiangr** [comment]
i' ve tried every windowed/borderless setting...it seems that the game doesn't like it though..even if i have 180 fps, the game feels choppy as if it had 50 fps when it's not in exclusive fullscreen

**Comment by Weltraumeule** [comment]
I'm searching for fullscreen Twitch overlay so long. The only app that works is the discontinued "Twitch Studio". However, it doesn't work with every game. And it is "over the top", as it is a streaming app like obs. I need only a chat app :D
Hope that the creator of Transparent Twitch Chat Overlay will digitally sign his app someday...

**Comment by Weltraumeule** [comment]
I will not play in borderless window, because always have less fps then in fullscreen mode :(
